internals/review_models.py

```diff
@@ -141,6 +141,7 @@ class Comment(ndb.Model):
   author = ndb.StringProperty()
   content = ndb.StringProperty()
   deleted_by = ndb.StringProperty()
+  migrated = ndb.BooleanProperty()
   # If the user set an approval value, we capture that here so that we can
   # display a change log.  This could be generalized to a list of separate
   # Amendment entities, but that complexity is not needed yet.
```

Comment:
Comment entities also contain an `old_approval_state` and `new_approval_state`. Should we ignore these or do they need to be migrated over as well?

Reply:
Good point.  Ideally, we would migrate over that early approach to storing history to Amendments. 

The amendments could have these values:
* field_name: just use the string "vote"
* old_value: the string from Approval.APPROVAL_VALUES[old_approval_state]
* new_value: the string from Approval.APPROVAL_VALUES[new_approval_state]

Reply:
It turns out that these values are currently all null, so no need to convert.